build_deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="malware-detection-frontend"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/malware-detection-frontend"
```

Comment:
It looks like you have not created a quay repo for the image. When you are creating, remember that the repo name should match here.